sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -98,6 +98,11 @@ export class EventHubConsumerClient {
     subscribe(partitionId: string, handlers: SubscriptionEventHandlers, options?: SubscribeOptions): Subscription;
     }
 
+// @public
+export interface EventHubConsumerClientOptions extends EventHubClientOptions {
+    allowDirectPartitionConnections?: boolean;
```

Comment:
Just curious - is it "allow" or "enable"? Ie, once this is on is it possible they might not get direct partition connections?

Reply:
If for some reason the event hub didn't support link redirect (not sure if this is ever the case), then we would not get a direct partition connection because we would not get a LinkRedirectError.